components/Map/Map.tsx

```diff
@@ -0,0 +1,215 @@
+import MapView, { PROVIDER_GOOGLE } from "react-native-maps";
+import { StyleSheet, View, Dimensions } from "react-native";
+
+const mapStyle = [
```

Comment:
Can you move this to the constants folder? just do "map.js" and inside  "export default mapStyle ... "

Reply:
do you want me to add the stylesheet to the const folders too or just the mapStyle?

Reply:
I think stylesheets are fine, and only move the map one if you agree with me but it's just pretty massive so I thought maybe it'd deserve it's own file